nautobot/core/events/__init__.py

```diff
@@ -55,7 +100,10 @@ def publish_event(*, topic, payload):
             lowest common denominator for serializability.
     """
     for event_broker in _EVENT_BROKERS:
-        event_broker.publish(topic=topic, payload=payload)
+        exclude_topics = event_broker.exclude_topics
+        include_topics = event_broker.include_topics
+        if not is_topic_match(topic, exclude_topics) and is_topic_match(topic, include_topics):
```

Comment:
If logically equivalent, this might be more intuitive:

```suggestion
        if is_topic_match(topic, include_topics) and not is_topic_match(topic, exclude_topics):
```